src/app/shared/header/header.component.html

```diff
@@ -0,0 +1,35 @@
+<mat-toolbar>
+  <mat-toolbar-row>
+    <button mat-icon-button (click)="toggleSideBar()">
+      <mat-icon>menu</mat-icon>
+    </button>
+
+    <span> DeskDine </span>
+
+    <div fxFlex fxLayout="row" fxLayoutAlign="flex-end">
+      <ul fxLayout="row" fxLayoutGap="20px" style="align-items: center">
+        <!-- <li>
+          <button mat-icon-button>
+            <mat-icon>settings</mat-icon>
+          </button>
+        </li> -->
+        <!-- <li>
+          <button mat-icon-button>
+            <mat-icon>help_outline</mat-icon>
+          </button>
+        </li> -->
```

Comment:
Now here, in real time project there is concept call TODO, if you commenting code and if you reason for that, we have to write 
//TODO your name and reason